tests/integration/cattletest/core/test_projects.py

```diff
@@ -372,6 +372,28 @@ def test_project_deactivate(user_clients, project, members):
     assert project.state == 'inactive'
 
 
+def test_project_member_invalid(project, admin_user_client, members):
+    client = client_for_project(project, admin_user_client)
+    project.setmembers(members=members)
+    diff_members(members, get_plain_members(project.projectMembers()))
+    membersGot = get_plain_members(project.projectMembers())
+    newMembers = []
+    oldMembers = []
+    for member in membersGot:
+        if (member['role'] == 'owner'):
+            newMembers.append(member)
+        else:
+            oldMembers.append(member)
+    project.setmembers(members=newMembers)
+    diff_members(newMembers, get_plain_members(project.projectMembers()))
+    with pytest.raises(ApiError) as e:
+        client.by_id('projectMember', 'garbageId')
+    assert e.value.error.status == 404
```

Comment:
This assert should be indented so that it is executed as part of the with. Same below.